guides/common/modules/con_understanding-package-mode-and-image-mode-hosts.adoc

```diff
@@ -0,0 +1,31 @@
+[id="understanding_package_mode_and_image_mode_hosts_{context}"]
+= Understanding Package mode and Image mode hosts
+
+In the latest update, hosts can now operate in two modes: package mode and image mode.
+
+.Image mode hosts
+Image mode hosts include eight new fields that reflect their boot status.
+These fields track the container images used in various scenarios:
+
+* *Booted Image & Digest*: Indicates the container image the host is currently using.
+* *Staged Image & Digest*: Indicates the container image the host uses after the next reboot.
+* *Available Image & Digest*: Represents the container image cached by running `Bootc upgrade --check`.
+* *Rollback Image & Digest*: Represents the container image the host reverts to after the next reboot if `Bootc rollback` is applied.
+
+.All host listing page updates
+* *New Column for Host Types*: The host listing page includes a column with icons that indicate the host type.
+** Clicking an icon displays the image path and digest.
+** Note that the image path can be used directly with podman or docker to pull the image from a registry.
+* *Search Enhancements*:
+** Use `image_mode=true` or `image_mode=false` to filter hosts by type.
+** Unregistered systems are assumed to be in package mode.
+** Search using eight new fields:
+`Bootc_booted_image`, `Bootc_booted_digest`, `Bootc_staged_image`, `Bootc_staged_digest`, `Bootc_available_image`, `Bootc_available_digest`, `Bootc_rollback_image`, `Bootc_rollback_digest`.
+
+.Host Details Page Enhancements
+A new card appears on the Details tab for registered hosts, displaying the eight fields from `Bootc`.
+
+.Important Notes
+* *Pull-Mode Remote Execution*: For pull-mode remote execution users, related packages must be included in the image mode host's container image.
```

Comment:
Not blocking your PR:

@ianballou Does that mean _all_ packages that are not present on a default RHEL base OS should be installed manually? I think this could also affect `subscription-manager` which ATIX packages for Debian/Ubuntu and SLES: https://oss.atix.de/ If it does, then it's probably even more packages such as OpenSCAP client and more.

Do you think that some packages such as `katello-host-tools-tracer` are not necessary anymore for container-based hosts because you'd rather rebuild+redeploy instead of updating?

Reply:
`subscription-manager` at least is available by default for Enterprise Linux systems, but you're right that any packages the user may need should get baked into the image. Others can be installed transiently after running `bootc usr-overlay`, but that is generally not recommended since it creates drift.

`subscription-manager` is crucial for the image mode feature since it is responsible for sending the bootc facts.

You're right about Tracer not being very useful for image mode hosts unless you are using the `bootc usr-overlay`.